erisdb/pipe/transactor.go

```diff
@@ -212,92 +212,6 @@ func (this *transactor) TransactAndHold(privKey, address, data []byte, gasLimit,
 	return ret, rErr
 }
 
-func (this *transactor) Send(privKey, toAddress []byte, amount int64) (*Receipt, error) {
```

Comment:
Weird, I just implemented the Send and SendAndHold methods in eris-db.js to support them at Tyler's request.